website/versioned_docs/version-0.7.0/userDocs/submarine-sdk/pysubmarine/development.md

```diff
@@ -24,7 +24,7 @@ This is required for developing & testing changes, we recommend installing pysub
 in its own conda environment by running the following
 
 ```bash
-conda create --name submarine-dev python=3.6
+conda create --name submarine-dev python=3.7
```

Comment:
I think the python codes with existing version documentation (0.6/0.7) can be left unchanged. Our focus can be more on master (0.8 or future).

Reply:
Thank you for the feedback.
I didn't notice that this document is under the v0.0.7 directory!